.travis.yml

```diff
@@ -1,49 +1,67 @@
+  - secure: qqJvtEJTq5DVimN/jFt8kuhIFVLFhG/hmx3iyLzQIhhBQqExLm+eURQj+lP/sjH3C+Z1yAg/bhaRw8FgZ+Cb4glMVU2XMDoxGlqoCsulrM5GMbKg4RwysiFCZhoOQ1fUmcfYnghtSHdMXFaIKTTEtN4KLzS4UGYPlMhfO8PBKv1EMXK8Eu6tTjXBQCVvnF6K3LbUJEuo7HdzCPwdGYmZvrncD51zoBZZZ38s0KuNwpP+mFtVub4h2yABgpWzM0O/xDfXw7i2tD2JwIOFqRlSzZnD0ymI0t3ArJoGJTbXnCBoKzOhR/vZKIMVflm9NzXzXLWqK3wtE+wHtQnnKKvuB5EAHP/og9e1SIM3oDmYpo5jffFkk1KIbt+6alnjzlFa56LKIuFeob2XWwqzkqfmHHojHSOdwZvxAESo6fy0d69jJiqPY3JFVXp2/oygsn0iUy4QH+28RcK9OdTkJV8jVdON7OVLQg5ZfAFIY4/prPdoQD/r3HVLIGmo856X+opdslwS3tu5KpqqbO5k17SGRl8wMCOY8+wPqyvrUuSr6kR8cDZuPS9dX0lvGAt9z323W732GiX2EwTFc19oldcTTzUTJPjnfaUcfDnuaobxvY4aoPLUy0Wc6+512S7YhoNi5lpkGKYuXNXAqMYODOULqZOTHCDCSOi9ALLbfjgjHWY=
+  - secure: oiDoRDBR9RZJlb1IrGlaszUpRU3cQNpQSlfo/oR2THL9s003seFvXYLiBvmwzrkYFsymXv+IfVk1IpDI1XoFQdbipCF0IWQMB3EUr35dwhzi7tLlrPbA1ypTNGriH+NbVobBKINlrjjWs7iI1pGw65oX62rjeocPkg8zENGEnRU0X+R4Y12k9i6sy5PedzzKhSFe+gBCjKwFVattPM1tEDNFlm1M6JWyanrhQHHOHKVvfA1EvH/qWcgWVebuPKFaExZp1ehwM7HSD30P8x7ois/83oem8lVkUrbCLVrktLd+2zg0j6lZAqrRR6mw+L/dtCK6g3Uxyl07GpCpE2cB/ChREhmnubFQuH6mJqWQ/Nh8lmusuPiiGPwgv2Yl0yLPmLf6NmmI6AuWLCTQ3xJ3I5eyNaiVsxYcsWiXpWCSZgvVeqVWpV1ghdssezvR5BoG5rkVxoSX73SWAIc7+hhkDPr5Y+gU23zwArxMpxpff9B8zL9s/sm7oTlugBRpT3dVg8JhCaywrstBZkO964JK5cGl4COKc1tHQ4Sh5lbOmATMAoI0lloOvhPdLC8sh/UxLemc4Bi4RgIeQD/LSZWpv11hk5+GJu+w/HvXrcbbCnMcHMvqf0E+4R1i8oRjZiVvGiNewS+y/B1L9pK14lUqA9UjCSPXdfHx7zy083xrhqw=
+  - secure: M49564mTJz/2VIft81hTeZ3H19rdpKd86hFG7gmdoaw/SE2WlNMTDln+tnVpp+mI8GTGnpjqPu9OgKiq0OZa+maj7jvo4zKHYzQZs73Db+FbvanUD0Nb9IBDCh+9K4ZxClloDv1aQEA0bvxBxJ4P4YdPfYWA8rzgkocrEtquVZ5QafpoD0yMNbMSetKbhbVHRreRqcTNY7JIZCBdCb7pW9SMQ0NJfDw6pxYXZXynkzTtBjGWb60M7g9hs0vrpCSQD3VP1CvGx7/35KIY62A52NP4JC5Z1+BM6OL2KCx78ZF6DUbt2Gyiq/qLdXOAfM9qEsKFDATr61kERpngyUd8be1JPpXlbl7afTiQmnIcGBwUqs3iwrhs2u1K2uSJGn0vR5NsdC3qoZWe24QpY1e2yKWXrloRKa0DaArDvTY/iyf+CAy/QxZHezO/IxWHYGhNLui3o5ZqR5wsvx/ZB2x6faXmtXHaR8LPm07/OZefF0k7/exyrDbkePJ0fYrHGzi3fJSLN58Lvn6i4CV5rxiRSOFsbK+uFlywd3mv7NNGvY/kArfOV7n/jsexP5L+KWEHvnZndB7dnqHnNTpCLVzpAORNmhRwakm9qoBBErYQJq1keZT7h55TIs5No2hbGgm075VecmZCXxgSLon71CZnD/07W6gkXmDXBlAjaMdRkOs=
+  - secure: aBsroS68VlT54LRD2fpX9s+d+B3Z779fitzJq5dRejFLBfGDpNUCdXXRRNqcxTe9ReQmDARFfp4yVGp8mmLLfVsMlCRAQMWZuI0KYukGgdRY31b84OUmM2hGLtgtiDGSCvVVhzHMrMXp15SFW6l2SYFWgv6Y+aPwfX7MVN63k3Fd7DrGN/u/nkLgOT+FHVyMw0E2GDBtUo1YOt1d6On6U83aZmnXQyVDa6MvyNa8gQHWiqQ+6lT74UMufzU/6Kuo9gh7VXD5evW3H3oyfONHw8L3acjhpH+8JMhrxCPjdCAaTF2fa2jyHDLl/SKjleW+Z5JbtT8XuReHyr/apEN7bj9DgMQ+dHsptuQoBaDCrX9pxAmrXt5vXoiCKT2uD569Ac5VjgKdd8Svuy4wNgSya3axO7eGa8mPrl6B4iOwMikhVdNvOAWdHRYXJHX6ccdM20vtM85Gtf6hQfryFvSHKhltqPRmA9wFhaMfBL0RsHbkI2P/hZSaYVi9W5lW3GdNA1VI/KGKlNCOLey5cbeFKRbm4O1c61AsHgrcWhx6qzWhTw17jdiAT9uQWiYqUtDxUTQGP4v6ZlpNEIkdasnJWPbQldBlJ4vT0D+XtqHMvUA4/gy3iMIofTl/wIxYs+C3ol+UE0e4xeM7XvKPkxo+J3T+RJ4EYFnpnXW8ddyIVXs=
```

Comment:
@nikooo777 What are all of these `secure` keys?

Reply:
those are all the env vars that once resided on travis.